packages/desktop/src/renderer/components/Settings/Settings.stories.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { FC, useState } from 'react'
+import { ComponentStory, ComponentMeta } from '@storybook/react'
+
+import { withTheme } from '../../storybook/decorators'
+
+import SettingsComponent, { SettingsComponentProps } from './SettingsComponent'
+
+// import { About } from '../widgets/Settings/About'
+
+// import { NotificationsComponent } from './Tabs/Notifications/NotificationsComponent'
+// import { NotificationsOptions, NotificationsSounds } from '@quiet/state-manager'
+
+import { InviteComponent } from './Tabs/Invite/InviteComponent'
+
+import { LeaveCommunityComponent } from './Tabs/LeaveCommunity/LeaveCommunityComponent'
+import { Typography } from '@mui/material'
+
+const Template: ComponentStory<typeof SettingsComponent> = args => {
+  return <SettingsComponent {...args} />
+}
+
+export const Component = Template.bind({})
+
+const Dummy: FC = () => {
+  return <Typography>Dummy</Typography>
+}
+
+const Leave: FC = () => {
+  return (
+    <LeaveCommunityComponent
+      communityName={'Rockets'}
+      leaveCommunity={jest.fn()}
+      open={false}
+      handleClose={jest.fn()}
+    />
+  )
+}
+
+// const Notifications: FC = () => {
+//   return (
+//     <NotificationsComponent
```

Comment:
Once again yes and no... Ideally we'd have components for each tab in storybook, but unfortunately they refuse to render because we use some electron-specific methods inside them. I left it as one day we may want to do a little refactor and actually use it